### ISSUE TITLE
Cannot add additional field to Newsletter system configuration at desired position

### ISSUE THREAD

**Issue by vasilii-b** [magento/magento2#19418]
### Summary (*)

**Environment**: Magento 2.2-develop, Magento 2.3-develop
As a Magento developer, I want to add some additional settings to the Newsletter system configuration before *Success Email Template* (as example).

### Examples (*)
Within a custom module, in the `adminhtml/system.xml` file, add a new field before *Success Email Template*, like following:

```
<?xml version="1.0"?>
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
    <system>
        <section id="newsletter">
            <tab>customer</tab>
            <group id="subscription">
                <field id="custom_field" translate="label comment" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
                    <label>Do some more stuff after subscription ?</label>
                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                </field>
            </group>
        </section>
    </system>
</config>
```

**Actual results**
With given above, the custom field will appear at the end of fields list in scope of Newsletter system configuration.

**Expected result**
Newly added field should be sorted according to given `sortOrder` value.

**The issue*
The problem is that in the `app/code/Magento/Newsletter/etc/adminhtml/system.xml` group fields has `sortOrder` the same - 1.

### Proposed solution
In the `app/code/Magento/Newsletter/etc/adminhtml/system.xml` group fields has `sortOrder` with a step of 10. So each additional field will get previous `sortOrder` value + 10.

**Comment by magento-engcom-team** [comment]
Hi @vasilii-b. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@vasilii-b do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @eduard13. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
@eduard13 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-96671`, `MAGETWO-96672` were created

**Comment by magento-engcom-team** [comment]
Hi @vasilii-b. Thank you for your report. 
The issue has been fixed in magento/magento2#19419 by @vasilii-b in 2.3-develop branch
Related commit(s):
 - [5937d865c4d6f358ac6ff66cc6217275f293e416](https://github.com/magento/magento2/commit/5937d865c4d6f358ac6ff66cc6217275f293e416)

The fix will be available with the upcoming 2.3.1 release.

**Comment by magento-engcom-team** [comment]
Hi @vasilii-b. Thank you for your report. 
The issue has been fixed in magento/magento2#19568 by @vasilii-b in 2.2-develop branch
Related commit(s):
 - [1a01f4ec48932c9dd3bdc3e7b0645902eff2196d](https://github.com/magento/magento2/commit/1a01f4ec48932c9dd3bdc3e7b0645902eff2196d)

The fix will be available with the upcoming 2.2.8 release.
